test/max-one-break-per-loop.js

```diff
@@ -0,0 +1,79 @@
+/**
+ * @fileoverview Tests for max-one-break-per-loop rule
+ * @author Artem Litchmanov <artem.litchmanov@gmail.com>
+ */
+
+'use strict';
+
+var Solium = require ('solium');
+
+var userConfig = {
+    "rules": {
+        "security/max-one-break-per-loop": "error"
+    }
+};
+
+describe ('[RULE] max-one-break-per-loop: Acceptances', function () {
+
+    it ('should accept all loops that have one or less breaks in them', function (done) {
+        var code = [
+            'function foo() { for(uint i = 0; i<10; i++) {break;} }',
```

Comment:
You need to wrap the code inside a contract declaration, otherwise the parser fails to parse the code.
`contract Foo { function bar() { /* Your code to test */ } }`

After modifying, please run tests & see if all passes. (If having trouble running tests, just let me know and I'll help you out or check out http://solium.readthedocs.io/en/latest/developer-guide.html#testing-your-plugin)